main.js

```diff
@@ -65,6 +65,11 @@ async function downloadCLI(version, platform, arch) {
     core.info(`⬇️ Downloading UpCloud CLI from ${downloadUrl}`);
     
     const downloadPath = await tc.downloadTool(downloadUrl);
+    if (false) { // TODO if >= v3.16.0
+      core.info('🔍 Verifying download');
+      await verifyDownload(downloadPath);
```

Comment:
Could use try-catch to print nicer error message in case verifying the attestations fails:

```suggestion
      try {
        await verifyDownload(downloadPath);
      } catch {
        throw new Error(`Verifying UpCloud CLI artifact attestations failed`);
      }
      core.info("UpCloud CLI artifact attestations verified successfully");
      ```